app/src/main/java/com/android/streetworkapp/ui/utils/CustomDialog.kt

```diff
@@ -33,7 +33,8 @@ fun CustomDialog(
     title: String = "Your $tag",
     Content: @Composable () -> Unit = {},
     onSubmit: () -> Unit = {},
-    onDismiss: () -> Unit = {}
+    onDismiss: () -> Unit = {},
+    verbose: Boolean = true
```

Comment:
Here Sonar has also raised an issue about too many function parameters. At first sight, it's not really possible to reduce the number of parameters. What do you think, there's room for improvement or not?

Reply:
I understand your point, I do agree with the fact that it is not really reducible : the principle of the `Dialog` is to be able to fit any pop-up need. 

Therefore, removing `onDismiss` or `onSubmit` would lessen the value of such a class for the codebase. This is also why I put default functions to simplify the usage as much as possible.

Also removing the `Content` function would prevent us from customizing the current pop-ups as we want such as **the rating of parks**, which uses a specific component to display interactible stars to rate, not simply text.